css/css-flexbox/reference/percentage-descendant-of-anonymous-flex-item-ref.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: percentage heights in descendants of anonymous flex items</title>
+<link href="resources/flexbox.css" rel="stylesheet">
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com">
+<button style="width: 200px; height: 100px;">
+    <div style="width: 200px; height: 100%; background-color: green;"></div>
```

Comment:
I'd directly use 100px, rather than depend on how that size is resolved here.

Reply:
Oh sure.